vllm/model_executor/models/minicpmv.py

```diff
@@ -216,16 +216,7 @@ def __init__(
 
         self.query = nn.Parameter(torch.zeros(self.num_queries, embed_dim))
         trunc_normal_(self.query, std=0.02)
-
-        if kv_dim is not None and kv_dim != embed_dim:
-            self.kv_proj = ReplicatedLinear(kv_dim, embed_dim, bias=False)
-        else:
-            # Maintain the same return value with ReplicatedLinear.forward
-            self.kv_proj = lambda *args, **kwargs: (
-                nn.Identity()(*args, **kwargs),
-                None,
-            )
```

Comment:
Could you explain why you made this change?

Reply:
If I haven't tested incorrectly, in models 2.0, 2.5, and 2.6, this layer is an instance of ReplicatedLinear.

Reply:
I think we should not assume that `kv_dim != embed_dim`. Even if this is true for the official HF repo, those who train a custom version of the model might have different configs.

Reply:
How should we modify it?

Reply:
I think the original logic should work fine.

Reply:
Fine, I will revert this snippet